server/src/main/java/keywhiz/service/permissions/Action.java

```diff
@@ -0,0 +1,23 @@
+package keywhiz.service.permissions;
+
+/**
+ * Action class defines the set of actions that can be performed on an object
+ * */
+public final class Action {
```

Comment:
Typically I'm very anti-comment but given that action is a very generic term I think this is helpful.